openedx/features/course_experience/templates/course_experience/latest-update-fragment.html

```diff
@@ -0,0 +1,24 @@
+## mako
+
+<%page expression_filter="h"/>
+<%namespace name='static' file='../static_content.html'/>
+
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML
+%>
+
+<%block name="content">
+<div class="update-message">
+    <h3>${_("Latest Update")}</h3>
+    <div class="dismiss-message">
+        <button type="button" class="btn-link">${_("Dismiss")}</button>
+    </div>
+
+    ${HTML(update_html)}
+</div>
+</%block>
+
+<%static:webpack entry="LatestUpdate">
+    new LatestUpdate();
```

Comment:
Nit: Andy was pushing for passing in the class names on this page as opposed to hard coding them in the .js file - see CourseHome.js for an example of how to pass it in through the 'options' object.